index.js

```diff
@@ -331,12 +332,15 @@ wss.on('connection', async function connection (client) {
   })
 })
 
-server.listen(port, config.hostname, (error) => {
-  if (error) {
-
-  }
-  logger.info(`HTTP Server listening on ${config.hostname} port ${port}`)
-})
+// only listen once the DB is sync'd
+db.sync().then( () => {
```

Comment:
Are we sure we want to do this here? It doesn't seem we were previously running `db.sync` in the index file, which is the API's entry point. @xlexi can probably provide more insight, but I think this may be the wrong approach.

Reply:
the reason you were not running it in index.js before was because it was getting automatically run in api/db/index.js the first time it was required - this was the source of my problem with bin/createTestUser.js

Reply:
And to explain - that piece of code is making sure the DB is fully sync'd before we let the http instance start listening for incoming requests.  I think this is again another race condition as you could have an incoming request before the DB is fully sync'd - highly unlikely though, but I figured it's better to be safe